gammapy/modeling/tests/test_sampling.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from gammapy.datasets import Datasets
+from gammapy.modeling.sampling import run_mcmc, ln_uniform_prior
+import numpy as np
+from gammapy.utils.testing import requires_dependency
+
+def get_datasets():
```

Comment:
You might want to use a purest fixture here.
First you need to `import pytest`

then rewrite the definition of the function:
```
@pytest.fixture(scope="session")
def dataset():
```

then add as a argument for the test functions : `dataset`
i.e. `def test_lnprob(dataset)`
and 
`def test_runmcmc(dataset) `

Reply:
thanks, now I was able to use pytest fixtures !